Map/LightOverlay.gd

```diff
@@ -77,13 +77,14 @@ var room_circle_radius: int = 40
 func _draw_room_circle(room: RoomUI):
 	var center_point_with_offset: Vector2 = Vector2(room.get_center_X() - offset_position_x, room.get_center_Y() - offset_position_y)
 	if room.get_light_level() == Enums.LightLevel.DIMLY_LIT:
-		draw_circle(center_point_with_offset, room_circle_radius, Color(0, 0, 0, 0.5))
+		draw_arc(center_point_with_offset, room_circle_radius, 0, TAU, 20, Color(0, 0, 0, 0.1), 3)
```

Comment:
what are the new parameters you added ?

Reply:
Left a comment to help explain

Reply:
I see the explanation, but why the new TAU parameter and the 20 ?

Reply:
Ah gotcha, that parameter for TAU is the end angle that we want to finish drawing the arc. TAU is a constant in Godot that basically gives the fully circumference of a circle. 20 is just an arbitrary number to draw the arc out. The more the points, the smoother the line. I figured I just keep it a small-ish number to not overload on the amount of points drawn.